### PR TITLE
Workingbranch4

### PR DESCRIPTION
**Pull request by jennymegan**
Final draft with new nav bar icon

### CODE REVIEW

File: indexstyle.css [hunk 1]
```diff
@@ -79,26 +79,41 @@ ul {
...
 }
 
+.thirdnav a {
+	border-left: 1px solid #A9A9A9;
+}
+
 li a:hover {
-	font-weight: 400;
+	text-shadow: 0 0 0.4px #000500;
+	transition-delay: 0.08s;
```
Comment (JoshThomasField) [inline indexstyle.css:107]: Do you want the effect to fade away as well or just instantly disapear?

File: index.html [hunk 2]
```diff
@@ -14,7 +14,8 @@
 		<ul>
 			<li><a href="#home">HOME</a></li>
 			<li><a href="#portfolio">PORTFOLIO</a></li>
-			<li><a href="#aboutme">ABOUT ME</a></li>
+			<li class="logo lastnav"><img src="logo1.png" alt="logo"/></li>
+			<li class="thirdnav"><a href="#aboutme">ABOUT ME</a></li>
 			<li class="lastnav"><a href="#contact">CONTACT</a></li>
```
Comment (JoshThomasField) [inline index.html]: Readability is easier with 
<li>
      <a></a>
</li>

**Review by JoshThomasField — CHANGES_REQUESTED** [review]

File: indexstyle.css (hunk 1 above)
Comment (jennymegan) [reply indexstyle.css:107]: ok with instant disappearance

File: index.html (hunk 2 above)
Comment (jennymegan) [reply index.html]: have indented as per your suggestion!

**Review by JoshThomasField — APPROVED** [review]

**Pull request merged**